lib/index.js

```diff
@@ -10,10 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* globals process, console, Q, JSON, escape, verifyToken, crypto */
+/* globals process, console, SDK_VERSION */
 /*jshint maxcomplexity:15 */
 
-var SDK_VERSION = '<%= sdkversion %>'; // Will be replaced by grunt
+require('./vendor/polyfills');
```

Comment:
I can add base64 from npm, but tree-shaking only works on ES6 imports (and webpack 2)